bitex/api/REST/gateio.py

```diff
@@ -0,0 +1,66 @@
+"""Gate.io REST API backend.
+
+Documentation available here:
+    https://gateio.io/api2
+"""
+# Import Built-ins
+import logging
+import hashlib
+import hmac
+# Import Third-Party
+
+# Import Homebrew
+from bitex.api.REST import RESTAPI
+from bitex.exceptions import IncompleteCredentialsError
+
+log = logging.getLogger(__name__)
+
+
+def getsign(params, secret):
```

Comment:
Keep the signing in the `sign_request_kwargs` method, as that's what it's meant for and this function gets used nowhere else.

Reply:
maybe copy from demo. How about put getsign into sign_request_kwargs?

Reply:
Since that method isn't used anywhere else, I dont see the benefit of putting it into a separate function.